include/seqan3/range/views/as_const.hpp

```diff
@@ -16,11 +16,13 @@
 
 #include <seqan3/utility/type_traits/function_traits.hpp>
 
+#ifdef SEQAN3_DEPRECATED_310
+
 namespace seqan3::detail
 {
 
 //!\brief Function object for seqan3::views::as_const.
-struct as_const_fn
+struct as_const_fn_deprecated
```

Comment:
Because you do not want to include `concatenated_sequences`, right?

Reply:
yes